src/components/PlaylistPage/MobileComponents.js

```diff
@@ -28,7 +28,7 @@ const MobileComponents = React.createClass({
 
             <Collapse in={this.state.showFilter}>
               <div>
-                <LessonFilter {...this.props}/>
+                <LessonFilter/>
```

Comment:
...og så kaller vi LessonFilterContainer, så blir det helt opplagt for dem som leser koden. (alle 4 stedene, selvsagt)